smash/io/hdf5_io.py

```diff
@@ -0,0 +1,1041 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from smash.core.model import Model
+
+from smash.core._constant import STRUCTURE_PARAMETERS, STRUCTURE_STATES
+
+from smash.io._error import ReadHDF5MethodError
+
+
+from smash.solver._mwd_setup import SetupDT
+from smash.solver._mwd_mesh import MeshDT
+from smash.solver._mwd_input_data import Input_DataDT
+from smash.solver._mwd_parameters import ParametersDT
+from smash.solver._mwd_states import StatesDT
+from smash.solver._mwd_output import OutputDT
+
+from smash.core._build_model import _build_mesh
+
+
+
+import os
+import errno
+import warnings
+import h5py
+import numpy as np
+import pandas as pd
+import smash
+
+
+
+__all__ = ["save_object_to_hdf5", "save_dict_to_hdf5", "save_smash_model_to_hdf5", "load_hdf5_file", "read_object_as_dict"]
```

Comment:
Faut que ce soit les memes que dans le fichier \_\_init\_\_.py A priori ca devrait etre ca
```suggestion
__all__ = ["save_smash_model_to_hdf5, load_hdf5_file, generate_smash_object_structure"]
```